atlp-rwanda/e-comm-team-techsmith-bn
# #184821383  Removing DS_Store from the repository

## What does this PR do?

Removes `DS_Store` from the repository and prevents git from tracking the file(s).

## Description of Task to be completed?

Deleting existing `DS_Store` from the repository and adding the file(s) in .gitignore. This prevents git from tracking it in the future.

## How should this be manually tested?

The changes are verifiable on this [link](https://github.com/atlp-rwanda/e-comm-team-techsmith-bn/compare/develop...bg-ignore-dsstore-184821383?expand=1).

Additionally, after merging this PR, we should notice that DS_Store will no longer be in the develop branch, and other branches as we update.

## What are the relevant pivotal tracker/Trello stories?

1. [#184821383](https://www.pivotaltracker.com/story/show/184821383)
